app/assets/stylesheets/css3/_transition.scss

```diff
@@ -58,9 +58,7 @@
 }
 
 @mixin transition-property ($properties...) {
-   -webkit-transition-property: transition-property-names($properties, 'webkit');
-      -moz-transition-property: transition-property-names($properties, 'moz');
-           transition-property: transition-property-names($properties, false);
+  @include prefixer(transform-property, $properties, webkit moz spec);
```

Comment:
I almost put up a PR to up the browser support the other day. I'm gonna do that now.